packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -27,6 +27,18 @@ export class MigrationActions {
     return this.ctx.migration.initialize()
   }
 
+  async renameSpecsFolder () {
+    if (!this.ctx.currentProject) {
+      throw Error('Need to set currentProject before you can rename specs folder')
+    }
+
+    const projectRoot = this.ctx.path.join(this.ctx.currentProject)
+    const from = path.join(projectRoot, 'cypress/integration')
+    const to = path.join(projectRoot, 'cypress/e2e')
```

Comment:
so the file paths are correct on windows
```suggestion
    const to = path.join(projectRoot, 'cypress', 'e2e')
```